diff --git a/dist/browser/lib/PEX.js b/dist/browser/lib/PEX.js
index b749cf35e6962380cee8671bd6c5f033ce9dcb78..b6db8459009e0b00e1328b7a103886b0edb6891e 100644
--- a/dist/browser/lib/PEX.js
+++ b/dist/browser/lib/PEX.js
@@ -183,7 +183,7 @@ class PEX {
                 // aud MUST be set by the signer or provided by e.g. SIOP/OpenID4VP lib
                 payload: {
                     iat: new Date().getTime(),
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             return Object.assign(Object.assign({}, decoded), { kbJwt });
@@ -338,7 +338,7 @@ class PEX {
                     payload: {
                         iat: new Date().getTime(),
                         nonce: proofOptions === null || proofOptions === void 0 ? void 0 : proofOptions.nonce,
-                        _sd_hash: sdHash,
+                        sd_hash: sdHash,
                     },
                 };
                 presentation = Object.assign(Object.assign({}, presentation), { kbJwt });
diff --git a/dist/main/lib/PEX.js b/dist/main/lib/PEX.js
index b749cf35e6962380cee8671bd6c5f033ce9dcb78..b6db8459009e0b00e1328b7a103886b0edb6891e 100644
--- a/dist/main/lib/PEX.js
+++ b/dist/main/lib/PEX.js
@@ -183,7 +183,7 @@ class PEX {
                 // aud MUST be set by the signer or provided by e.g. SIOP/OpenID4VP lib
                 payload: {
                     iat: new Date().getTime(),
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             return Object.assign(Object.assign({}, decoded), { kbJwt });
@@ -338,7 +338,7 @@ class PEX {
                     payload: {
                         iat: new Date().getTime(),
                         nonce: proofOptions === null || proofOptions === void 0 ? void 0 : proofOptions.nonce,
-                        _sd_hash: sdHash,
+                        sd_hash: sdHash,
                     },
                 };
                 presentation = Object.assign(Object.assign({}, presentation), { kbJwt });
diff --git a/dist/module/lib/PEX.js b/dist/module/lib/PEX.js
index 65aa74adf86973e99345938c826613ddcba6ca7e..464545177f8d156ba07019e20e0e6c17e6d086ef 100644
--- a/dist/module/lib/PEX.js
+++ b/dist/module/lib/PEX.js
@@ -174,7 +174,7 @@ export class PEX {
                 // aud MUST be set by the signer or provided by e.g. SIOP/OpenID4VP lib
                 payload: {
                     iat: new Date().getTime(),
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             return {
@@ -339,7 +339,7 @@ export class PEX {
                 payload: {
                     iat: new Date().getTime(),
                     nonce: proofOptions?.nonce,
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             presentation = {
