diff --git a/dist/index.js b/dist/index.js
index d1f2d7555fe6503e4cbef92358f458f6f91a9da3..067d51132530d030f1d74c3bb19ff561c54d3c82 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -27,8 +27,8 @@ var __exportStar = (this && this.__exportStar) || function(m, exports) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.jsonErrorHandler = exports.sendErrorResponse = void 0;
-const dotenv = __importStar(require("dotenv-flow"));
-dotenv.config();
+// const dotenv = __importStar(require("dotenv-flow"));
+// dotenv.config();
 __exportStar(require("./entra-id-auth"), exports);
 __exportStar(require("./static-bearer-auth"), exports);
 __exportStar(require("./auth-utils"), exports);
diff --git a/src/index.ts b/src/index.ts
index 1f6bb2fdcae60d3f6408595a1d5757057c92a78d..0acae13058090c54e498b827c8fa221df5d9a97e 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -1,10 +1,10 @@
-import * as dotenv from 'dotenv-flow'
-dotenv.config()
-export * from './entra-id-auth'
-export * from './static-bearer-auth'
-export * from './auth-utils'
-export * from './express-builders'
-export * from './types'
-export { sendErrorResponse, jsonErrorHandler } from './express-utils'
-export * from './functions'
-export * from './openid-connect-rp'
+// import * as dotenv from 'dotenv-flow'
+// dotenv.config()
+export * from './entra-id-auth';
+export * from './static-bearer-auth';
+export * from './auth-utils';
+export * from './express-builders';
+export * from './types';
+export { sendErrorResponse, jsonErrorHandler } from './express-utils';
+export * from './functions';
+export * from './openid-connect-rp';
