diff --git a/dist/tokens/index.js b/dist/tokens/index.js
index 66c4a3d4341593056d318adefe5652a371bd989a..6f6ca930e57dbd6f05f1d8bbf50dcabf4b13ffa4 100644
--- a/dist/tokens/index.js
+++ b/dist/tokens/index.js
@@ -117,7 +117,7 @@ const assertValidAccessTokenRequest = (request, opts) => __awaiter(void 0, void
                 throw new oid4vci_common_1.TokenError(400, oid4vci_common_1.TokenErrorResponse.invalid_grant, `${oid4vci_common_1.PIN_VALIDATION_ERROR} ${txCodeOffer.length}`);
             }
         }
-        if (request.tx_code !== credentialOfferSession.txCode) {
+        if (request.tx_code !== credentialOfferSession.userPin) {
             throw new oid4vci_common_1.TokenError(400, oid4vci_common_1.TokenErrorResponse.invalid_grant, oid4vci_common_1.PIN_NOT_MATCH_ERROR);
         }
     }
