diff --git a/dist/browser/lib/PEX.js b/dist/browser/lib/PEX.js
index b749cf35e6962380cee8671bd6c5f033ce9dcb78..b6db8459009e0b00e1328b7a103886b0edb6891e 100644
--- a/dist/browser/lib/PEX.js
+++ b/dist/browser/lib/PEX.js
@@ -183,7 +183,7 @@ class PEX {
                 // aud MUST be set by the signer or provided by e.g. SIOP/OpenID4VP lib
                 payload: {
                     iat: new Date().getTime(),
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             return Object.assign(Object.assign({}, decoded), { kbJwt });
@@ -338,7 +338,7 @@ class PEX {
                     payload: {
                         iat: new Date().getTime(),
                         nonce: proofOptions === null || proofOptions === void 0 ? void 0 : proofOptions.nonce,
-                        _sd_hash: sdHash,
+                        sd_hash: sdHash,
                     },
                 };
                 presentation = Object.assign(Object.assign({}, presentation), { kbJwt });
diff --git a/dist/main/lib/PEX.js b/dist/main/lib/PEX.js
index b749cf35e6962380cee8671bd6c5f033ce9dcb78..b6db8459009e0b00e1328b7a103886b0edb6891e 100644
--- a/dist/main/lib/PEX.js
+++ b/dist/main/lib/PEX.js
@@ -183,7 +183,7 @@ class PEX {
                 // aud MUST be set by the signer or provided by e.g. SIOP/OpenID4VP lib
                 payload: {
                     iat: new Date().getTime(),
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             return Object.assign(Object.assign({}, decoded), { kbJwt });
@@ -338,7 +338,7 @@ class PEX {
                     payload: {
                         iat: new Date().getTime(),
                         nonce: proofOptions === null || proofOptions === void 0 ? void 0 : proofOptions.nonce,
-                        _sd_hash: sdHash,
+                        sd_hash: sdHash,
                     },
                 };
                 presentation = Object.assign(Object.assign({}, presentation), { kbJwt });
diff --git a/dist/module/lib/PEX.js b/dist/module/lib/PEX.js
index 65aa74adf86973e99345938c826613ddcba6ca7e..b963ec86360ac1d861583031fbea56d1d8f40731 100644
--- a/dist/module/lib/PEX.js
+++ b/dist/module/lib/PEX.js
@@ -3,7 +3,7 @@ import { Status } from './ConstraintUtils';
 import { EvaluationClientWrapper } from './evaluation';
 import { PresentationSubmissionLocation, } from './signing';
 import { PEVersion, SSITypesBuilder } from './types';
-import { calculateSdHash, definitionVersionDiscovery, getSubjectIdsAsString } from './utils';
+import { calculatesd_hash: sdHash, definitionVersionDiscovery, getSubjectIdsAsString } from './utils';
 import { PresentationDefinitionV1VB, PresentationDefinitionV2VB, PresentationSubmissionVB, ValidationEngine } from './validation';
 /**
  * This is the main interfacing class to be used by developers using the PEX library.
@@ -174,7 +174,7 @@ export class PEX {
                 // aud MUST be set by the signer or provided by e.g. SIOP/OpenID4VP lib
                 payload: {
                     iat: new Date().getTime(),
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             return {
@@ -339,7 +339,7 @@ export class PEX {
                 payload: {
                     iat: new Date().getTime(),
                     nonce: proofOptions?.nonce,
-                    _sd_hash: sdHash,
+                    sd_hash: sdHash,
                 },
             };
             presentation = {
